contest/templates/contest/contest.html

```diff
@@ -111,22 +111,35 @@ <h1 class="panel-title" style="text-align:center;">Clarification</h1>
     <!--Contest detail-->
     <div id="detail">
       <ul class="nav nav-tabs" role="tablist" id="contest_tab">
-        <li class="active"><a href="#overview" role="tab" data-toggle="tab">Overview</a>
+        <li class="active" role="presentation"><a href="#overview" role="tab" data-toggle="tab">Overview</a>
         </li>
-        <li><a href="#problem" role="tab" data-toggle="tab">Problem</a>
+        <li class="dropdown" role="presentation">
+          <a href="#problem" class="dropdown-toggle" data-toggle="dropdown">
+            Problem<span class="caret"></span>
```

Comment:
the tag itself is a icon
http://getbootstrap.com/components/#glyphicons-examples